src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -717,7 +738,34 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             );
         }
 
-        this.setState({ feedItems, activityFeedError: undefined });
+        const baseChanges = { feedItems, activityFeedError: undefined };
+
+        // If activeFeedEntryId is not set or
```

Comment:
This comment is largely describing the logic below. We should use comments to explain "why" something is written if it's not obvious in the code.